app/src/ui/cherry-pick/cherry-pick-flow.tsx

```diff
@@ -154,7 +159,7 @@ export class CherryPickFlow extends React.Component<ICherryPickFlowProps> {
             openFileInExternalEditor={openFileInExternalEditor}
             resolvedExternalEditor={resolvedExternalEditor}
             openRepositoryInShell={openRepositoryInShell}
-            sourceBranchName={sourceBranch.name}
+            sourceBranchName={sourceBranch ? sourceBranch.name : null}
```

Comment:
I think Markus preferred to avoid truthy/falsey checks:
```suggestion
            sourceBranchName={sourceBranch !== null ? sourceBranch.name : null}
```

Reply:
Yep. Sorry. in grained habit thanks for the catch.